package.json

```diff
@@ -23,7 +27,9 @@
     "vinyl-source-stream": "^1.1.0",
     "watchify": "^3.7.0"
   },
-  "scripts": {},
+  "scripts": {
+    "test": "karma start --single-run --browsers PhantomJS"
```

Comment:
We should make another script for running gulp. We won't be making new builds every now and then. I prefer not to add it to `npm test`. We should make `npm build` or something which is a good way to do it.